src/web/calculators/linear/components/LinearCalculator.tsx

```diff
@@ -30,6 +30,8 @@ import {
   calculateUnloadedSpeed,
 } from "web/calculators/linear/linearMath";
 import { useLinearWorker } from "web/calculators/workers";
+import { nominalVoltage } from "../../../../common/models/Motor";
+import { calculateKa, calculateKg, calculateKv } from "../../shared/sharedMath";
```

Comment:
nit (here and other places) - use absolute imports. this can be configured in vscode to auto use absolute imports if you want https://stackoverflow.com/questions/52432191/auto-import-in-visual-studio-code-only-offering-absolute-path-with-lerna-subpack